test/utils.jl

```diff
@@ -80,34 +77,25 @@ module TestUtils
             3rd Quartile:   3.250000
             Maximum:        4.000000
             Length:         4
-            Type:           $Int
+            Type:           Int64
 
             nullarr
             Summary Stats:
-            Mean:           3.500000
-            Minimum:        2.000000
-            1st Quartile:   2.750000
-            Median:         3.500000
-            3rd Quartile:   4.250000
-            Maximum:        5.000000
             Length:         4
-            Type:           $Int
-            Number Missing: 0
-            % Missing:      0.000000
+            Type:           Union{Int64, Nulls.Null}
```

Comment:
Running through these tests on 0.7-DEV now has these `Union{T, Null}`s printing as `?T` so I don't think we need to bother with https://github.com/JuliaData/DataTables.jl/pull/68 anymore.